tracekit.js

```diff
@@ -248,6 +262,38 @@ TraceKit.report = (function reportModuleWrapper() {
         _onErrorHandlerInstalled = true;
     }
 
+    /**
+     * Uninstall the global onerror handler
+     * @memberof TraceKit.report
+     */
+    function uninstallGlobalHandler() {
+        window.onerror = _oldOnerrorHandler;
+        _onErrorHandlerInstalled = false;
+    }
+
+    /**
+     * Install a global onunhandledrejection handler
+     * @memberof TraceKit.report
+     */
+    function installGlobalUnhandledRejectionHandler() {
+        if (_onUnhandledRejectionHandlerInstalled === true) {
+            return;
+        }
+
+        _oldOnunhandledrejectionHandler = window.onunhandledrejection;
+        window.onunhandledrejection = traceKitWindowOnUnhandledRejection;
+        _onUnhandledRejectionHandlerInstalled = true;
+    }
+
+    /**
+     * Uninstall the global onunhandledrejection handler
+     * @memberof TraceKit.report
+     */
+    function uninstallGlobalUnhandledRejectionHandler() {
+        window.onerror = _oldOnunhandledrejectionHandler;
```

Comment:
We should check to see if the handler is installed first, otherwise we could be setting someone else's handler to undefined.